engine/crates/engine-v2/src/http_response/mod.rs

```diff
@@ -72,22 +61,16 @@ impl HttpGraphqlResponse {
         )
     }
 
-    pub(crate) fn build(
-        response: Response,
-        format: Option<StreamingFormat>,
-        metadata: HttpGraphqlResponseExtraMetadata,
-    ) -> Self {
-        let mut http_response = if let Some(format) = format {
+    pub(crate) fn build(response: Response, format: Option<StreamingFormat>) -> Self {
```

Comment:
:bug: **Bug**
The `build` function no longer initializes the `metadata` field of `HttpGraphqlResponse`. This could lead to issues if other parts of the codebase expect this field to be initialized. Since the `metadata` field is removed, ensure that all usages of `HttpGraphqlResponse` are updated accordingly to not rely on this field.

```suggestion
// Ensure that all usages of HttpGraphqlResponse are updated to not rely on the metadata field.
```